client/state/selectors/get-recommended-plugins.js

```diff
@@ -0,0 +1,16 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns a list of recommended plugins for the given site.
+ *
+ * @param  {Object}    state 		Global state tree
+ * @param  {Number}    siteId 	The ID of the site we're querying
+ * @return {?Array<Object>} 		Array of plugin objects
```

Comment:
This suggests we might be returning a `null`, but it seems we'll always be returning an array (albeit an empty one).